blocks/floating-images/floating-images.css

```diff
@@ -0,0 +1,88 @@
+/* styles for the floating images auto block */
+
+.block.image-right,
+.block.image-left {
+  margin-bottom: 5em;
```

Comment:
do em unites make sense for images?  There's no text so this could be unpredictable.